### DISCUSSION TITLE
Libre 3 + xDrip (Web Follower as hardware data source) unavailable

### DISCUSSION THREAD

**Discussion by ikimi66** [NightscoutFoundation/xDrip#2414] [Q&A]
Hi, there, Does Anyone know why my xdrip+ could not use web follower to get libre3 librelinkup data, unavailable from the day before yesterday (10/26). The log shows collector service due to lack of reception: backoff:120. What's the problem?
I've also updated the latest xdrip+ still get the same issue. 

![313345552_10224660512199354_2331975220752716791_n](https://user-images.githubusercontent.com/103375139/199387289-7a2059dd-f538-4956-aedb-1ade687df920.jpg)

**Comment by ikimi66** [comment]
Using the latest version 2022.11.02
Get "no permission to obtain location"
Does anyone encounter the same issue as me?


![72855](https://user-images.githubusercontent.com/103375139/199701107-344fab93-b885-407e-82f5-51fb69b4a037.jpg)
![72856](https://user-images.githubusercontent.com/103375139/199701117-9d805aef-df9a-41c0-bec2-2afe3dcfdb44.jpg)

**Comment by ikimi66** [comment]
reinstall xdrip+, still not available

**Reply by fabricejoubert** [reply]
Looks like I got this message before. Try to connect with the same account to the official "libre link up" application. It will be necessary to validate new conditions and should unblock the situation.
